src/CattyTests/Parser & Serialization/CatrobatLanguage0.991/XMLParserTests/Unit Tests/XMLParserTests0991.swift

```diff
@@ -537,6 +537,15 @@ class XMLParserTests0991: XMLAbstractTest {
         XCTAssertTrue(whenTouchDownScript.isKind(of: WhenTouchDownScript.self), "Invalid script type")
     }
 
+    func testWhenBackgroundChangedScript() {
+        let project = self.getProjectForXML(xmlFile: "ValidProjectAllBricks0991")
```

Comment:
Can you please move this script from _ValidProjectAllBricks0991_ to _ValidProjectAllBricks0992_ and also to `XMLParserTests0992.swift`?

Reply:
Sure, done. :)